### PR TITLE
PPTT-4733 Updated HTTP API `ec_products` description, optional parameters

### PR DESCRIPTION
**Pull request by sebastianpiskorski**

### CODE REVIEW

File: data_collection/api/tracking_api.yaml [hunk 1]
```diff
@@ -458,21 +458,23 @@ paths:
...
               - [5] brand: `string` (optional) a brand of the product
               - [6] variant: `string` (optional) a variant of the product
-              - [7] product custom dimensions: `Object<string, string>` a dictionary of custom dimension values, numeric string ID and a string with value. Max 20 product custom dimensions, 20 is max ID.
+              - [7] product custom dimensions: `Object<string, string>` (optional) a dictionary of custom dimension values, numeric string ID and a string with value. Max 20 product custom dimensions, 20 is max ID.
 
-            Legend: [index] name: `data type` (requisite) Desctiption. Restrictions.
+            Legend: [index] name: `data type` (requisite) Description. Restrictions.
 
-            E-commerce cart must not contain more than 100 products.
+            E-commerce cart must not contain more than 100 products.
```
Comment (wronan) [inline data_collection/api/tracking_api.yaml:477]:
```suggestion
            E-commerce cart must not contain more than 100 products.
```

**Review by wronan — APPROVED** [review]

**Review by pmajocha — APPROVED** [review]

**Comment by kororokke** [comment]
18.8?

**Comment by sebastianpiskorski** [comment]
Yes, 18.8, last week.

**Comment by kororokke** [comment]
Last week was 18.18 and not 18.8. And it was already deployed. So... 18.18?

**Comment by sebastianpiskorski** [comment]
Sorry, I was too hasty. Fixed, and now it is `18.18` and should I do the PR to the `18.19` as well?

**Comment by kororokke** [comment]
I’ll clean it up tomorrow morning

**Review by kororokke — APPROVED** [review]

**Comment by kororokke** [comment]
@sebastianpiskorski Cleaned up. All good.

**Pull request merged**